app/controllers/report_controller/widgets.rb

```diff
@@ -655,7 +655,17 @@ def widget_set_record_vars(widget)
     #      end
     #      widget.miq_widget_shortcuts = ws
     #    end
-    widget.options[:url] = @edit[:new][:url] unless @edit[:new][:url].blank?
+    if @sb[:wtype] == "rf"
```

Comment:
Great, yes, I agree.  Fix the bug for now... I honestly didn't review all of the surrounding code to see where it's coming from.  Feel free to say, that's outside the scope of this PR and we can look at that later.